{
  "uid" : "83f9b0d29c04ba28",
  "name" : "User manually selects a different language -- @1.1 ",
  "fullName" : "Language Selection and Translation: User manually selects a different language",
  "historyId" : "f5d7d04c21620617d981f49fa8f1bd70",
  "time" : {
    "start" : 1746711076917,
    "stop" : 1746711076951,
    "duration" : 34
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: STEP: Given the system supports the following languages\n",
  "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_012_steps.py\", line 14, in step_given_supported_languages\n    raise AssertionError(msg)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: STEP: Given the system supports the following languages\n",
    "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_012_steps.py\", line 14, in step_given_supported_languages\n    raise AssertionError(msg)\n",
    "steps" : [ {
      "name" : "Given the system supports the following languages:",
      "time" : {
        "start" : 1746711076919,
        "stop" : 1746711076919,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: STEP: Given the system supports the following languages\n",
      "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_012_steps.py\", line 14, in step_given_supported_languages\n    raise AssertionError(msg)\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "9d83f2a33863073d",
        "name" : ".table",
        "source" : "9d83f2a33863073d.csv",
        "type" : "text/csv",
        "size" : 31
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "Given a logged-in user",
      "time" : {
        "start" : 1746711076951,
        "stop" : 1746711076951,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When the user selects English from the language settings",
      "time" : {
        "start" : 1746711076951,
        "stop" : 1746711076951,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then the system should display the UI in English",
      "time" : {
        "start" : 1746711076951,
        "stop" : 1746711076951,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Language Selection and Translation"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "language",
    "value" : "English"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 3,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 3
      },
      "items" : [ {
        "uid" : "b64e10939fe69d3e",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/21/#testresult/b64e10939fe69d3e",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: Given the system supports the following languages\n",
        "time" : {
          "start" : 1746710242620,
          "stop" : 1746710242653,
          "duration" : 33
        }
      }, {
        "uid" : "21d3d50599209466",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/20/#testresult/21d3d50599209466",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: Given the system supports the following languages\n",
        "time" : {
          "start" : 1746709788738,
          "stop" : 1746709788772,
          "duration" : 34
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "83f9b0d29c04ba28.json",
  "parameterValues" : [ "English" ]
}